server/komodo-rest/src/main/java/org/komodo/rest/relational/json/VirtualizationStatusSerializer.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * See the COPYRIGHT.txt file distributed with this work for information
+ * regarding copyright ownership.  Some portions may be licensed
+ * to Red Hat, Inc. under one or more contributor license agreements.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301 USA.
+ */
+package org.komodo.rest.relational.json;
+
+import static org.komodo.rest.Messages.Error.UNEXPECTED_JSON_TOKEN;
+import java.io.IOException;
+import org.komodo.rest.Messages;
+import org.komodo.rest.relational.response.virtualization.RestVirtualizationStatus;
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * A GSON serializer/deserializer for {@status RestTeiidVdbStatusVdb}s.
+ */
+public final class VirtualizationStatusSerializer extends TypeAdapter< RestVirtualizationStatus > {
```

Comment:
Same here as KEntity, if not stored in the repository there is no requirement for custom serializer?

Reply:
See above.

Reply:
got it, thanks confused over KRestEnity over KAbstractEntiry. Maybe a better name is KRespositoryEntity and ReponseEnitity, WDYT?

Reply:
Certainly can consider this.